srcf/database/compat.py

```diff
@@ -83,6 +84,8 @@ def joindate(self):
         return self.joined.strftime("%Y/%m")
 
 # in the old API, admins was a method
```

Comment:
I think this comment needs to live on the line directly above `class AdminsSetCompat`, so swap it with the two lines of spacing inserted.